threema-connector-demo/src_hd/threema/connector/demo/multipleRecipients/multipleRecipients.xhtml

```diff
@@ -0,0 +1,37 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:ic="http://ivyteam.ch/jsf/component" xmlns:p="http://primefaces.org/ui"
+  xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+  <ui:composition template="/layouts/basic-10.xhtml">
```

Comment:
all dialogs should be built upon the latest 'frame' template: 
... to get it simply create a dummy dialog, on the creation wizard pick the frame template, ... then delete dialog (but the template is created in /webcontent) ... so you can change it everywhere where the 'basic' template was referenced.

Reply:
Changed template to latest frame-template